src/components/journeys/editor-journey/editor-journey.jsx

```diff
@@ -0,0 +1,450 @@
+                            {
+                                imgSrc: '/images/onboarding-journeys/Fly-Animation.jpg',
+                                text: intl.formatMessage(messages.flyAnimationTitle),
+                                handleOnClick: () => driverObj.moveTo(7)
+                            }
+                        ]}
+                    />
+                }
+            }
+        },
+        {
+            popover: {
+                title: intl.formatMessage(messages.createTitle),
+                showButtons: ['close'],
+                sectionComponents: {
+                    description: <EditorJourneyDescription
+                        title={intl.formatMessage(messages.typeTitle)}
+                        descriptionData={[
+                            {
+                                imgSrc: '/images/onboarding-journeys/Record-Music.jpg',
+                                text: intl.formatMessage(messages.recordSoundTitle),
+                                handleOnClick: () => driverObj.moveTo(8)
+                            },
+                            {
+                                imgSrc: '/images/onboarding-journeys/Make-Music.jpg',
+                                text: intl.formatMessage(messages.makeMusicTitle),
+                                handleOnClick: () => driverObj.moveTo(9)
+                            }
+                        ]}
+                    />
+                }
+            }
+        },
+        {
+            popover: {
+                title: intl.formatMessage(messages.createTitle),
+                showButtons: ['close'],
+                sectionComponents: {
+                    description: <EditorJourneyDescription
+                        title={intl.formatMessage(messages.startTitle)}
+                        descriptionData={[
+                            {
+                                imgSrc: '/images/onboarding-journeys/Tutorials-Icon.png',
+                                text: intl.formatMessage(messages.tutorialTitle),
+                                handleOnClick: () => openTutorial(onActivateDeck, tutorials.clicker, driverObj)
+                            },
+                            {
+                                imgSrc: '/images/onboarding-journeys/Starter-Projects-Icon.png',
+                                text: intl.formatMessage(messages.starterProjectTitle),
+                                handleOnClick: () => redirectToProject(projects.clicker)
+                            },
+                            {
+                                imgSrc: '/images/onboarding-journeys/On-Own-Icon.png',
+                                text: intl.formatMessage(messages.onMyOwnTitle),
+                                handleOnClick: () => ownOptingPicked(setCanViewTutorialsHighlight, driverObj)
+                            }
+                        ]}
+                    />
+                }
+            }
+        },
```

Comment:
Is it worth to extract this to a function with `tutorialId` and `projectId` parameters, instead of having 6 almost identical copies?

Reply:
Hm yeah, I was thinking of separating the data for the steps and having a builder to iterate over it but it was going to be the same. But this logic is totally extractable